common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleIntegrationSuite.java

```diff
@@ -86,6 +85,17 @@ public class ExternalShuffleIntegrationSuite {
     new byte[54321],
   };
 
+  private static TransportConf createTransportConf(String maxRetries, String rddEnabled) {
```

Comment:
nit: specify using the actual types and convert it to `String` in this method.

```suggestion
  private static TransportConf createTransportConf(int maxRetries, boolean rddEnabled) {
```